tests/integration/docker/user.bats

```diff
@@ -45,3 +45,22 @@ setup() {
 	[ "${status}" -eq 0 ]
 	[[ "${output}" == 'root'* ]]
 }
+
+@test "Run with additional groups" {
+	run $DOCKER_EXE run --rm --group-add audio --group-add nogroup busybox id
```

Comment:
Gotcha. In which case, it would be good to have a test that asserts the expected failure if a non-existent group is specified using `--group-add`.